plugins/leemons-plugin-assignables/frontend/src/components/ActivityHeader/index.js

```diff
@@ -95,6 +104,10 @@ export default function ActivityHeader({
 
   const { classes } = useTotalLayoutStyles();
 
+  const goToAssignmentDetail = () => {
+    history.push(`/private/assignables/details/${instance?.id}`);
+  };
```

Comment:
It shouldn't be roleDetails.dashboardURL ?? `/private/assignables/details/:id` with the replacement of :id?

To match the behavior of the other views